recipes/configs/llama3_3/70B_full.yaml

```diff
@@ -107,8 +109,7 @@ dtype: bf16
 # Logging
 metric_logger:
   _component_: torchtune.training.metric_logging.DiskLogger
-  log_dir: ${output_dir}
-output_dir: /tmp/full-llama3_3-finetune
+  log_dir: ${output_dir}/logs
```

Comment:
I think this is not consistent with other configs? Maybe update all of the rest configs?

Reply:
The others were updated yesterday :)